src/translate_rapid_pro_keys.py

```diff
@@ -113,8 +126,12 @@ def remap_radio_shows(cls, user, data, coda_input_dir):
         :param coda_input_dir: Directory to read coded coda files from.
         :type coda_input_dir: str
         """
-        # No implementation needed yet, because no flow is yet to go wrong in production.
-        pass
+        # Correct drama week 6 messages recovered from the Hormuud downtime issue
+        cls._remap_radio_show_by_time_range(
+            user, data, "received_on", "rqa_s01e06_raw",
+            isoparse("2019-05-27T00:00:25+03:00"),
```

Comment:
Is there someone who could double check these dates?

Reply:
Sorted the csv by date picked the minimum and maximum dates. The log on number of remapped messages matches the no of loaded messages. Episode 6 dates double checked with RDA.